distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Pattern.enso

```diff
@@ -97,7 +99,7 @@ type Pattern
 
        Arguments:
        - input: The text to match the pattern described by `self` against.
```

Comment:
This could use a brief comment explaining the special handling of zero-length matches.